apps/base/utils.py

```diff
@@ -138,6 +138,17 @@ def send_email(
     return
 
 
+def get_aws_secret_keys():
+    aws_keys = {
+        "AWS_ACCOUNT_ID": os.environ.get("AWS_ACCOUNT_ID"),
```

Comment:
what if the environment variable is not set? you have to pass a default value too right?

Reply:
@Ram81 I've changed this to how RIshabh suggested. Haven't pushed because I'm waiting for confirmation on some other comments.